tests/testsuite/alt_registry.rs

```diff
@@ -1492,8 +1494,10 @@ dependencies = [
 [[package]]
 name = "foo"
 version = "0.1.0"
-source = "sparse+http://[..]/"
-checksum = "458c1addb23fde7dfbca0410afdbcc0086f96197281ec304d9e0e10def3cb899""#,
+source = "sparse+http://127.0.0.1:[..]/index/"
```

Comment:
It seems that `source = "source = "sparse+http://[..]/index/"` also passes the test.

Reply:
Yes.  I had snapshot updating process all of these and re-added the globbing.  I tried to be more limited in where I added globs.  I also tried to find where we could replace globs with something else (e.g. the `[BROKEN_PIPE]`) so snapshot updating can be more automatic.

I could possibly detect localhost with a port and redact the port (`127.0.0.1:[PORT]`).  The main risk with redactions is we'd redact something that another test explicitly checks (e.g. most of the time we don't care about the `.crate` size reported by `cargo publish` but some tests specifically do)

Reply:
Got it. Make sense. Thanks!